cryptofeed/standards.py

```diff
@@ -174,6 +177,7 @@ def timestamp_normalize(exchange, ts):
         BINANCE_US: 'ticker',
         BINANCE_JERSEY: 'ticker',
         BINANCE_FUTURES: UNSUPPORTED,
+        BLOCKCHAIN: 'ticker',
```

Comment:
Since bid/ask prices aren't provided by the feed, just leave this as `UNSUPPORTED` to avoid confusion.

Reply:
yeah - good point 👍